test/batchverify.test.js

```diff
@@ -0,0 +1,252 @@
+//           26073464383897998325899031212762184271676052677226679463708862316754828477519n,
+//           20246927599389923510374971105736264637524117420538179767629249587300902801762n,
+//           1n,
+//           7867784340861643381702890578607277776011430152424699121581244161773093676488n,
+//         ], [
+//           44370630452101398467924320556439323233869374150616132215183787387252341590058n,
+//           38448863731492799660668882834560725606410712239157980760146247592118262650597n,
+//           1n,
+//           28837567775264640306185442010835055372505962142426868564781195691916034475874n,
+//         ]
+//       ];
+//       const A = [
+//         16962727616734173323702303146057009569815335830970791807500022961899349823996n,
+//         11903303657706407974989296177215005343713679411332034699907763981919547054807n
+//       ];
+//       const msg = [
+//         33455n,
+//         0n
+//       ];
+//       const R8 = [
+//         78142972218048021222160463610080218564159109753358461787358041591257467621730n,
+//         38448863731492799660668882834560725606410712239157980760146247592118262650597n
+//       ];
+//       const S = [
+//         4869643893319708471955165214975585939793846505679808910535986866633137979160n,
+//         5004556735901913393272427758925840403246877222315506387332009764265656498271n
+//       ];
+//       let bufMsg = [];
+//       let bufR8 = [];
+//       let bufS = [];
+//       let bufA = [];
+//       let bitsMsg = [];
+//       let bitsR8 = [];
+//       let bitsS = [];
+//       let bitsA = [];
+//       for (let i = 0; i < 2; i++) {
+//         bufMsg.push(utils.bigIntToLEBuffer(msg[i]));
+//         bufR8.push(utils.bigIntToLEBuffer(R8[i]));
+//         bufS.push(utils.bigIntToLEBuffer(S[i]));
+//         bufA.push(utils.bigIntToLEBuffer(A[i]));
+//       }
+//       for (let i = 0; i < 2; i++) {
+//         bitsMsg.push(utils.pad(utils.buffer2bits(bufMsg), 16));
+//         bitsR8.push(utils.pad(utils.buffer2bits(bufR8), 256));
+//         bitsS.push(utils.pad(utils.buffer2bits(bufS), 255).slice(0, 255));
+//         bitsA.push(utils.pad(utils.buffer2bits(bufA), 256));
+//       }
+//       const chunkA = [];
+//       const chunkR = [];
+//       for (let j = 0; j < 2; j++) {
+//         let chunkATemp = [];
+//         let chunkRTemp = [];
+//         for (let i = 0; i < 4; i++) {
+//           chunkATemp.push(utils.chunkBigInt(pointA[j][i]));
+//           chunkRTemp.push(utils.chunkBigInt(pointR[j][i]));
+//         }
+//         for (let i = 0; i < 4; i++) {
+//           utils.pad(chunkATemp[i], 5);
+//           utils.pad(chunkRTemp[i], 5);
+//         }
+//         chunkA.push(chunkATemp);
+//         chunkR.push(chunkRTemp);
+//       }
+//       const witness = await cir.calculateWitness({
+//         msg: bitsMsg, A: bitsA, R8: bitsR8, S: bitsS, PointA: chunkA, PointR: chunkR,
+//       }, true);
+
+//       assert.ok(witness[3] === 3n);
+//     });
+//   });
+// });
```

Comment:
Is there any reason to have a commented-out code? If it's deprecated, could you please remove it? If it's not, this should be rather disabled instead of commenting out.